### ISSUE TITLE
Публикация русской документации на офф. сайте

### ISSUE THREAD

**Issue by stfalcon** [avalanche123/symfony-docs-ru#57]
Всем привет.

Перевод итальянцев выложили на офф. сайт http://symfony.com/it/doc/current/book/index.html.
Хотелось бы увидеть там нашу русскоязычную доку.

@frost-nzcr4, @dbykadorov, @Skorney, @avalanche123, @helios-ag, что вы об этом думаете?

**Comment by inoryy** [comment]
До публикации пока что как до луны :)
Подробнее можно тут пообсуждать #56

**Comment by cursedcoder** [comment]
Думаю, что нужно это более широко развивать.
Русское комьюнити уже довольно большое, но оно не сплочено.
Думаю 90% даже не знает о существовании этого репа, а остальные подавно забыли.
Возможно, если каждый из нас отпостит в твиттерах\форумах пару строчек призыва, 
это хоть как-то поможет найти новых людей.

Ну а до этого, нужно решить основательные проблемы, такие как:
- ветки master и 2.0
- глоссарий
- старые переводы, их качество оставляет желать лучшего

Так-же я думал о каких-нибудь тулзах для перевода, искал в сети, но подходящего не нашёл.
В принципе можно накидать простенький редактор html+js с подсветкой строчек и общим скроллингом например.

**Comment by dbykadorov** [comment]
Я не против приложиться, но за пол года они там столько поменяли, что как бы не заново пришлось переводить. Я как-то мержил апдейт в главу о доктрине, проклял всё )

Конкретные предложения - нужны переводчики. В одиночку это никто не потянет. Я с прошлого года ещё не отошёл ))) Плюс нет никакого фидбека, не всё удачно переведено, есть опечатки и т.п.

**Comment by dbykadorov** [comment]
@cursedcoder поддерживаю, коммьюнити разрозненное. Я обращался к нескольким переводчикам в своё время, с просьбой коммитить, но они фигачили в свои блоги переводы.

**Comment by cursedcoder** [comment]
@dbykadorov согласен, есть опечатки, а так-же иногда опускается часть текста.
Про блоги переводчиков, думаю это в основном из-за неопытности в Git.

У меня есть желание переводить, может буду успевать 1 страничку в 1-2 дня, но всё же.
Я так понимаю, что @Inori тоже заинтересован в этом.

**Comment by stfalcon** [comment]
да. мержить обновления это головняк. думаю нужно хотя бы 2.0 ветку добить и выложить

**Comment by Skorney** [comment]
Сейчас проблема в том, что не удобно читать то что есть - трудно подправлять ошибки. Нужно продумать оптимальное решение для чтения репозитория в формате html.

**Comment by cursedcoder** [comment]
@Skorney можно собрать сэндбокс для этого, вкупе со Сфинксом и остальными примочками.

**Comment by stfalcon** [comment]
> Сейчас проблема в том, что не удобно читать то что есть - трудно подправлять ошибки. Нужно продумать оптимальное решение для чтения репозитория в формате html.

ок. давай попробуем развернуть на поддомене и настроить автоапдейт

**Comment by dbykadorov** [comment]
Ну у меня это всё есть, сфинкс, автоапдейт. Уже полгода мыслю форум сделать для обсуждения перевода ) http://symfony-gu.ru

**Comment by cursedcoder** [comment]
@dbykadorov это хорошо что у тебя есть, но я бы хотел и себе локально поставить, удобнее же имхо.

**Comment by dbykadorov** [comment]
И локально есть, там настроить пара пустяков. Питон, несколько эггов и верёд )

**Comment by Skorney** [comment]
А можно сделать на самом гитхабе сайт? как здесь: http://visionmedia.github.com/mocha/

**Comment by everzet** [comment]
@cursedcoder `easy_install -U Sphinx` ?

**Comment by cursedcoder** [comment]
@everzet суть не в этом, было бы куда лучше с единым сэндбоксом, который любой желающий может в любое время скачать с ГХ и поставить себе на машину.

**Comment by inoryy** [comment]
может, глупый вопрос, но есть ли какая-то система слежения за коммитами/апдейтами?
#38 читал, но так и не понял к чему вы пришли.

Если нет, может имеет смысл пообщаться с итальянскими коллегами на этот счет?

**Comment by zgoniaiko** [comment]
не против приложиться, естественно не в ущерб работе

**Comment by zgoniaiko** [comment]
словарь был для 1.4 где-то на траке. может часть его тут приживется

**Comment by dbykadorov** [comment]
@Inori если есть выход на них и нет языкового барьера - то да, было бы не плохо. Из своей же практики могу точно сказать - что поддерживать перевод в разы сложнее чем тупо его сделать один раз.

**Comment by cursedcoder** [comment]
@zgoniaiko не думаю, #55 вот сабж по этому

**Comment by helios-ag** [comment]
Также пришел к выводу, что и @everzet в бессмысленности перевода тех. документации. Поэтому более переводом заниматься скорее всего не буду.

**Comment by stfalcon** [comment]
но столько времени потратить на перевод и не довести до ума тоже как-то не true. я считаю, что версию для 2.0 нужно добить и опубликовать

**Comment by cursedcoder** [comment]
Да не стоит забивать, лишь из-за того, что пару человек высказались о бессмысленности перевода.
Это их личное мнение. Я уверен, что есть ещё очень много людей, которым это на самом деле понадобиться.

**Comment by dbykadorov** [comment]
Мотиватор с SymfonyLive'2012 http://www.slideshare.net/garak/translating-symfony-docs

**Comment by cursedcoder** [comment]
Общаюсь с капитаном команды турецких переводчиков. Ребята полностью перевели /book за 2 недели

**Comment by dbykadorov** [comment]
@cursedcoder и что он говорит? Перевести не такая проблема как актуализировать имхо

**Comment by cursedcoder** [comment]
@dbykadorov чтобы актуализировать, нужно сначала перевести

**Comment by dbykadorov** [comment]
@cursedcoder всё ещё хуже ) надо начать

**Comment by inoryy** [comment]
Не вижу смысла начинать пока у нас нет четких планов/возможностей поддерживать постоянные изменения.

Алсо у этого репозитория есть еще мейнтейнеры помимо @avalanche123 ? Если нет то было бы неплохо начать с этого

**Comment by dbykadorov** [comment]
Насколько я знаю нету. В общем начинать надо с людей. Кто готов заниматься переводом и поддержкой. Перекличку чтоли сделать... )

**Comment by everzet** [comment]
Ребят, окститесь, у вас 2 месяца ушло на вопрос "что делать". И через 2 месяца вы решили сделать перекличку. Да не интересно это никому кроме вас.

Как поюзать - так это все будут рады, а как контрибутить... Вообщем, оно того не стоит, лучше контрибутьте в офф. доки.

**Comment by frost-nzcr4** [comment]
Слегка утрируя, замечу, что программист, имеющий опыт работы и хотя бы базовые знания английского, в переводе документации не нуждается, а новичок с симфони может не справиться даже имея на руках переводы, т. е. до любого инструмента надо ещё дорасти, хотя хороший грамотный русский текст мог бы ускорить этот рост. Но судя по численности и активности группы мало кто в нём нуждается. Думаю, вопрос о переводах решится сам, когда появится спрос на них, а пока можно поконтрибутить во что-нибудь другое)

**Comment by dbykadorov** [comment]
Ну вот поэтому и сидим у разбитого корыта - толи от гордости великой, толи хер знает от чего ) Ладно, зовите если будут светлые мысли

**Comment by inoryy** [comment]
Ну может сначала все же сделать перекличку, а потом выводы?

Начинать что-то делать все равно нет смысла пока не
- Назначены мейнтейнерами более активные люди чем @avalanche123
- Продумана четкая система по мержам с обновлениями
- Составлен roadmap

**Comment by cursedcoder** [comment]
@Inori плюсую, но, видимо никто не хочет брать на себя лишнее, париться с роадмапами и системами.

**Comment by frost-nzcr4** [comment]
На счёт мэйнтейнеров - начните коммитить в свой форк, люди потянутся, это же гит. Опять же чёткую систему не придумать не начав коммитить. Роадмап, я думаю, лучше сначала подтянуть переведённые доки, а потом начать перевод новых.

**Comment by dbykadorov** [comment]
Есть мнение что гитхаб не лучшее место для координации ) Давайте попробуем скайп skype:?chat&blob=cmJfmMHYMyY-Exq4ErGprDVPTDsFWhH2XbnZ3j-0reLfuKxfz49LBWpFZw

Подтягивайтесь кому интересно
